lib/generator/command_line/generator_optparser.rb

```diff
@@ -3,7 +3,7 @@
 module Generator
   class GeneratorOptparser
     DEFAULT_OPTIONS = {
-      freeze: false,
+      freeze: true,
```

Comment:
Since `:freeze` is no longer an option, it would be good to be able to use `:update` as the options hash key.